guides/common/modules/proc_deleting-multiple-content-view-versions.adoc

```diff
@@ -0,0 +1,15 @@
+[id="Deleting_Multiple_Content_View_Versions_{context}"]
+= Deleting Multiple Content View Versions
+
+You can delete multiple Content View Versions simultaneously.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Content View*.
+. Click the Content View you want to delete versions of.
+. On the *Versions* tab, select the check box of the version or versions you want to delete. 
+. Click the vertical ellipsis icon at the top of the list of Content Views.
+. Click *Delete* to open the deletion wizard that shows any affected environments.
+. If there are no affected environments, review the details and click *Delete*. 
+. If there are any affected environments, the deletion wizard has the option to reassign any hosts or activation keys before deletion.
```

Comment:
@maximiliankolb would it be better if I changed this to "If there are any affected environments, reassign any hosts or activation keys"?

Reply:
Yes, that's a good idea.